components/camera/videosource/static.go

```diff
@@ -64,8 +66,8 @@ type fileSourceAttrs struct {
 	DistortionParameters *transform.BrownConrady            `json:"distortion_parameters,omitempty"`
 	Stream               string                             `json:"stream"`
```

Comment:
I think we are supposed to remove stream here. https://docs.google.com/document/d/18p3LV6ghH-m0xGjO7MZRz4zK0X8JePmGZRAXBtm0k7E/edit?disco=AAAAjtHiigc.

Reply:
My bad... done